### PR TITLE
move not running overlay to the supervisor

### PR DESCRIPTION
**Pull request by akosyakov**
#### How to test
- stop a workspace and check that loading screen is shown with stopped state and start button
- try to restart the workspace from the loading screen as well as from the dashboard, check that the tab with stopped workspace gets reloaded in both cases
- check that each workspace tab has only one connection to the gitpod server (if not reconnections happen)

#### Out of scope
- testing out of credit alerts
- moving credit notifications from Theia

### CODE REVIEW

**Comment by csweichel** [comment]
Maybe it would make sense to get started with react in the front-end. If would also make life easier when we push more functionality there.

File: components/dashboard/src/components/start-workspace.tsx [hunk 1]
```diff
@@ -330,11 +335,41 @@ export class StartWorkspace extends React.Component<StartWorkspaceProps, StartWo
...
+
+                    if (!stoppedReason.endsWith(".")) {
+                        stoppedReason += ".";
+                    }
+                    stoppedReason = <div>{stoppedReason}</div>;
+                }
+            }
+            let startButton;
+            if (this.state.workspaceInstance.status.phase === 'stopped') {
+                startButton = <Button className='button' variant='outlined' color='secondary' onClick={() =>
```
Comment (svenefftinge) [inline components/dashboard/src/components/start-workspace.tsx:360]: should be contained in `<div className="message action">`

File: components/dashboard/src/components/start-workspace.tsx [hunk 2]
```diff
@@ -354,19 +389,27 @@ export class StartWorkspace extends React.Component<StartWorkspaceProps, StartWo
...
+                message = <div className='message'>
+                    <div>Gitpod Credit Alert</div>
+                    <Button className='button' variant='outlined' color='secondary' onClick={() =>
+                        window.open(new GitpodHostUrl(window.location.toString()).asUpgradeSubscription().toString(), '_blank')
+                    }>Upgrade Subscription</Button>
+                </div>;
+            } else if (this.state.inTheiaAlready) {
                 message = <div className='message'></div>;
             } else {
                 this.ensureWorkspaceAuth(this.state.workspaceInstance.id)
```
Comment (svenefftinge) [inline components/dashboard/src/components/start-workspace.tsx:415]: in stopping and stopped we don'T want to see productivity tips.
I.e. see https://github.com/gitpod-io/gitpod/pull/1907/files#diff-9b675c748d29d1be3fb144e36f5b22ceR421
we should

**Review by svenefftinge — APPROVED** [review]

**Pull request merged**